mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -1,19 +1,79 @@
 pipeline {
     agent none
     stages {
+        // Workaround Durable Tasks bug, hopefully
+        stage("Library build") {
+            agent {
+                docker {
+                    image 'rocm/mlir:rocm4.1-latest'
+                    args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+                    label "rocm"
+                    alwaysPull true
+                }
+            }
+            stages {
+                stage('Build libMLIRMIOpen') {
+                    steps {
+                        checkout scm
+                        cmakeBuild generator: 'Ninja',\
+                            buildDir: 'build',\
+                            installation: 'InSearchPath',\
+                            buildType: 'Release',\
+                            steps: [[args: 'libMLIRMIOpen']],\
+                            cmakeArgs: '-DBUILD_FAT_LIBMLIRMIOPEN=ON'
+                        cmake arguments: '--install . --component libMLIRMIOpen',\
+                            installation: 'InSearchPath', workingDir: 'build'
+
+                    }
+                }
+                stage('Build MIOpenDriver') {
+                    steps {
+                        dir('MIOpen') {
+                            git branch: 'develop', poll: false,\
+                                url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
+                            cmake arguments: "-P install_deps.cmake --minimum",\
+                                installation: "InSearchPath"
+                            cmakeBuild generator: 'Ninja',\
```

Comment:
MIOpenDriver is not built with Ninja. I found it out because I copied this script to Jenkinsfile.nightly-perf, then MIOpenDriver got seg fault for a few Resnet50 configurations. I changed this piece of code back to 'make', and MIOpenDriver is working well now.